src/System.Security.Cryptography.Pkcs/src/netcore50/project.json

```diff
@@ -0,0 +1,26 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.Platforms": "1.0.1-rc3-24022-00",
+    "System.Collections": "4.0.10",
+    "System.Diagnostics.Contracts": "4.0.0",
+    "System.Diagnostics.Debug": "4.0.10",
+    "System.ObjectModel": "4.0.10",
+    "System.Resources.ResourceManager": "4.0.0",
+    "System.Runtime": "4.0.20",
+    "System.Runtime.Handles": "4.0.0",
+    "System.Runtime.InteropServices": "4.0.20",
+    "System.Runtime.WindowsRuntime": "4.0.10",
+    "System.Security.Cryptography.Encoding": "4.0.0-rc3-24022-00",
+    "System.Security.Cryptography.Primitives": "4.0.0-rc3-24022-00",
+    "System.Security.Cryptography.X509Certificates": "4.1.0-rc3-24022-00",
+    "System.Text.Encoding": "4.0.10",
+    "System.Threading": "4.0.10",
+  },
+  "frameworks": {
+    "netcore50": {}
```

Comment:
Was netcore50 being special needed, or just because you used X509 as a template?